.github/actions/bpftrace/scripts/check-encryption-leaks.bt

```diff
@@ -149,7 +147,7 @@ kprobe:br_forward
           (!$pod_to_pod_via_proxy && ($src_is_pod && $dst_is_pod) && (!$src_is_internal && !$dst_is_internal)) ||
           ($pod_to_pod_via_proxy && ($src_is_pod || $dst_is_pod))
         ) {
-        if ($ip4h->protocol != PROTO_TCP || !$tcph->rst) {
+        if ($ip4h->protocol != PROTO_TCP) {
```

Comment:
```suggestion
```

We need to remove the whole line here, otherwise we would enter and log only plain text non-TCP packets.
Same applies below.

Reply:
We don't run the bpftrace script in `conformance-gateway-api.yaml`, right?
Is `gatewayAPI` being tested in any of our e2e tests where we use the leak detection script?